packages/knip/src/plugins/vite/index.ts

```diff
@@ -10,7 +10,7 @@ const enablers = ['vite', 'vitest'];
 
 const isEnabled: IsPluginEnabled = ({ dependencies }) => hasDependency(dependencies, enablers);
 
-export const config = ['vite*.config.{js,mjs,ts,cjs,mts,cts}'];
+export const config = ['vite*.config.{js,mjs,ts,cjs,mts,cts}', 'src/vite-env.d.ts'];
```

Comment:
It should be an `entry` file (to be added as source file in the dependency graph), `config` files are meant to be loaded and parsed.

Reply:
Entry doesn't works in my project ;(
I'll try to understand why

Reply:
Maybe because it's a type definition file

Reply:
I found the reason.
1. In Vite plugin we already have `resolveEntryPaths` - https://github.com/webpro-nl/knip/blob/12d14c6bee72625eb1ded52301acd08585aa2943/packages/knip/src/plugins/vite/index.ts#L23
2. This `resolveEntryPaths` will return the entry path here - https://github.com/webpro-nl/knip/blob/12d14c6bee72625eb1ded52301acd08585aa2943/packages/knip/src/WorkspaceWorker.ts#L302
3. And it is taken by priority here, ignoring entries - https://github.com/webpro-nl/knip/blob/12d14c6bee72625eb1ded52301acd08585aa2943/packages/knip/src/util/plugin.ts#L76

I see two options to fix this.
1. to merge `entry` and result of `resolveEntryPaths` in `getFinalEntryPaths`, but I think thats not correct. 
2. customize `resolveEntryPaths` in Vite plugin description like
```ts
const viteResolveEntryPaths = async (localConfig, options) => {
    const vitestEntryPaths = await resolveEntryPaths(localConfig, options);
    return [...vitestEntryPaths, toEntryPattern('src/vite-env.d.ts')];
};

export default {
    title,
    enablers,
    isEnabled,
    config,
    production,
    resolveEntryPaths: viteResolveEntryPaths,
    resolveConfig,
};
```

And select second one. @webpro what do you think?